modules/rosa-classic-aws-rhcos-ami.adoc

```diff
@@ -0,0 +1,969 @@
+|AWS AMI
+|Description
+
+.19+|*us-east-1*
+|`ami-00bd5e71974ac1199`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-083b6a0085bfb83d7`
+|`rhcos-46.82.202008071940-0`
+|`ami-02325594e61f75251`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-0a7427bf6706eed45`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-0341d736b214ee026`
+|`rhcos-45.82.202008010929-0`
+|`ami-043dd22d4deee53b4`
+|`rhcos-49.84.202107191624-0`
+|`ami-043e9f42db2ddbb7d`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0405e95d69021031e`
+|`rhcos-45.81.202005200134-0`
+|`ami-06f3dda44132a8cc9`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-00c25def04737ae38`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-0a75974bb6c224b00`
+|`rhcos-47.83.202102090044-0`
+|`ami-04e0a14f518d73ed5`
+|`rhcos-48.84.202107202156-0`
+|`ami-0063c6510b80631ac`
+|`rhcos-46.82.202008181646-0`
+|`ami-0e2a30fbaf81ecdfa`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-094e791abb6450740`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-00354720d36d019f9`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-06fe61aa8989b20ef`
+|`rhcos-48.84.202105062123-0`
+|`ami-04af4cb679580f10d`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-042ac701257912c93`
+|`rhcos-42.80.20191002.0-hvm`
+
+.19+|*us-east-2*
+|`ami-0452b47871cf2ce52`
+|`rhcos-413.86.202301302144-0-x86_64`
+|`ami-029060ed19236eb1f`
+|`rhcos-48.84.202107202156-0`
+|`ami-08dca13891a09d1cf`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
+|`ami-04737283ac19b8635`
+|`rhcos-414.92.202305090606-0-x86_64`
+|`ami-01dfe6692d6973bc9`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-06bcf566958d44a2a`
+|`rhcos-46.82.202008071940-0`
+|`ami-0b38f74547a5aa475`
+|`rhcos-45.82.202008010929-0`
+|`ami-01d3fc6434b82a558`
+|`rhcos-416.94.202403071059-0-x86_64`
+|`ami-0d63475428237d0cd`
+|`rhcos-47.83.202102090044-0`
+|`ami-03780ec2f9f1a9502`
+|`rhcos-415.92.202309201557-0-x86_64`
+|`ami-0e677f92eb4180cc0`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-0475717ada8f8ab59`
+|`rhcos-46.82.202008181646-0`
+|`ami-0859369e0b9336804`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-0950fb52c5ea6e785`
+|`rhcos-45.81.202005200134-0`
+|`ami-0f086e071ab012ede`
+|`rhcos-410.84.202111201002-0-x86_64`
+|`ami-0f231a947c43e463d`
+|`rhcos-49.84.202107191624-0`
+|`ami-026682fa83d6aa548`
+|`rhcos-48.84.202105062123-0`
+|`ami-0c7f325b64d31da97`
+|`rhcos-411.85.202203242008-0-x86_64`
+|`ami-01f3aa6486e2fabc3`
+|`rhcos-44.82.202008011133-0-hvm`
+
+.19+|*us-west-1*
+|`ami-08d8c763a8104ee84`
+|`rhcos-47.83.202102090044-0`
+|`ami-0fb62bc12119cc013`
+|`rhcos-44.82.202008011133-0-hvm`
+|`ami-089a0db9eed618f9b`
+|`rhcos-45.81.202005200134-0`
+|`ami-004c0234e98ec95d9`
+|`rhcos-48.84.202107202156-0`
+|`ami-042eb05b65a167924`
+|`rhcos-42.80.20191002.0-hvm`
+|`ami-062d213892df037a9`
+|`rhcos-412.86.202207142104-0-x86_64`
+|`ami-00f21f28bd96234fb`
+|`rhcos-43.82.202008010953.0-hvm`
+|`ami-06124fcfba15a55f2`
+|`[Copied ami-05f09360cb1869045 from us-east-1] rhcos-44.81.202007071934-0-hvm`
```

Comment:
Can we clarify why the `[Copied ami-.... from us-east-1]` is there? Can we remove it?